### PR TITLE
Change to use common ASCII characters

### PR DESCRIPTION
**Pull request by ydah**
- Use U+002d instead of U+2013
- Use U+2019 instead of U+0060

______________________________________________________________________

Before submitting the PR make sure the following are checked:

- [x] Feature branch is up-to-date with `master` (if not - rebase it).
- [x] Squashed related commits together.
- [-] Added tests.
- [x] Updated documentation.
- [-] Added an entry to the `CHANGELOG.md` if the new code introduces user-observable changes.
- [x] The build (`bundle exec rake`) passes (be sure to run this locally, since it may produce updated documentation that you will need to commit).

### CODE REVIEW

**Review by pirj — APPROVED** [review]
Thanks!

**Comment by pirj** [comment]
> Use U+2019 instead of U+0060

Isn't it the other way around?

**Comment by bquorning** [comment]
Is this just because it’s simpler to use ASCII, or is there another reason? I kinda wish that we could use proper typography in our documentation, but I also know that it would probably be a larger project that I’m willing to fight for 😅

**Comment by pirj** [comment]
My main problem with typography in this context was Antora's search feature, e.g. `won't` wouldn't be found. But Algolia seems to have fixed this. Or I'm making this up due to my bad memory.
And I have no objections if it was displayed with typographic symbols, but the source is ASCII, and the search just works either way.

**Review by bquorning — APPROVED** [review]

**Comment by pirj** [comment]
Thank you!

File: docs/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -32,7 +32,7 @@ over
 calculator.compute(line_item).should == 5
 ----
 
-is a feature of RSpec itself - you can read about it in the https://relishapp.com/rspec/rspec-expectations/docs/syntax-configuration#disable-should-syntax[RSpec Documentation].
+is a feature of RSpec itself – you can read about it in the "Disable should syntax" section of https://rspec.info/features/3-12/rspec-expectations/syntax-configuration[RSpec Documentation].
```
Comment (pirj) [inline docs/modules/ROOT/pages/index.adoc]: Oh, I think I've messed up with this merge :facepalm:

**Pull request merged**